LayoutTests/fast/events/no-mousemove-on-modifier-key-press.html

```diff
@@ -0,0 +1,43 @@
+<html>
+<head>
+<script src="../../resources/js-test.js"></script>
+<script src="../../resources/ui-helper.js"></script>
+<style>
+.hidden {
+    display: none;
+}
+</style>
+</head>
+
+<body>
+<p id="description"></p>
+<p id="manual-instructions" class="hide">To run this test by hand, press a modifier key.</p>
+<div id="console"></div>
+<script>
+window.jsTestIsAsync = true;
+
+async function runTest()
+{
+    if (!window.testRunner)
+        document.getElementById("manual-instructions").classList.remove("hidden");
+
+    function fireKeyDownEventsWithModifiers() {
+        ['metaKey', 'altKey', 'shiftKey', 'ctrlKey'].forEach(modifierKey => {
+            UIHelper.keyDown("k", [modifierKey]);
+            debug('Dispatched KeyDown event with modifier ' + modifierKey);
+        });
+    }
+
+    window.addEventListener('mousemove', () => testFailed("MouseMove event detected"));
+
+    if (window.testRunner)
+        fireKeyDownEventsWithModifiers();
```

Comment:
We'll want to `await` here as well once we make the above change.

Reply:
Yup, asyncified the test in https://github.com/WebKit/WebKit/pull/14221/commits/c88025563e2436195514cb0c0a22883be08bf627